pkg/client/client.go

```diff
@@ -864,6 +889,59 @@ func (c *Client) WaitForPrometheus(ctx context.Context, p *monv1.Prometheus) err
 	return nil
 }
 
+func (c *Client) ValidatePrometheus(ctx context.Context, promNsName types.NamespacedName) error {
+	var validationError error
+
+	pollErr := wait.Poll(10*time.Second, 5*time.Minute, func() (bool, error) {
```

Comment:
is 10s poll too aggressive and 5m timeout too much :) ?

Reply:
This is 0.1 Qps, I think it is not significant traffic to the Prometheus.